src/core/Database/MysqliStatementProxy.php

```diff
@@ -17,6 +17,8 @@
 
 class MysqliStatementProxy extends ObjectProxy
 {
+    public const IO_METHOD_REGEX = '/^close|execute|fetch|prepare/i';
```

Comment:
> Regex is slow. Personally, I prefer to call `isset()` or `array_key_exists()` on an associative array instead of using regex.

it is a pity that PHP method name is not case sensitive, regex is the easiest way